### PR TITLE
Compressible and Boussinesq coriolis change

### PR DESCRIPTION
**Pull request by Witt-D**
A quick PR to change how the Coriolis force is defined in the compressible Euler and Boussinesq equations sets.

I have not got round to editing some testing scripts to ensure for this, I was hoping to do this after the linear solver PR so that it does not get held up

### CODE REVIEW

File: gusto/equations/compressible_euler_equations.py [hunk 1]
```diff
@@ -216,10 +216,12 @@ def __init__(self, domain, parameters, Omega=None, sponge_options=None,
         # -------------------------------------------------------------------- #
         # Extra Terms (Coriolis, Sponge, Diffusion and others)
         # -------------------------------------------------------------------- #
-        if Omega is not None:
-            # TODO: add linearisation
-            residual += coriolis(subject(prognostic(
-                inner(w, cross(2*Omega, u))*dx, "u"), self.X))
+        if parameters.Omega is not None:
+            # TODO: add linearis0ation
```
Comment (tommbendall) [inline gusto/equations/compressible_euler_equations.py:220]:
```suggestion
            # TODO: add linearisation
```

File: gusto/equations/shallow_water_equations.py [hunk 2]
```diff
@@ -165,13 +165,13 @@ def __init__(self, domain, parameters, fexpr=None, bexpr=None,
 
         if fexpr is not None:
             V = FunctionSpace(domain.mesh, 'CG', 1)
-            f = self.prescribed_fields('coriolis', V).interpolate(fexpr)
+            self.f = self.prescribed_fields('coriolis', V).interpolate(fexpr)
```
Comment (jshipton) [inline gusto/equations/shallow_water_equations.py:168]: You don't need to put `f` in `self` - you can access it later through `self.prescribed_fields('coriolis')`

File: gusto/equations/boussinesq_equations.py [hunk 3]
```diff
@@ -40,7 +40,6 @@ class BoussinesqEquations(PrognosticEquationSet):
 
     def __init__(self, domain, parameters,
                  compressible=True,
-                 Omega=None,
```
Comment (tommbendall) [inline gusto/equations/boussinesq_equations.py:43]: You also need to remove the argument from the docstrings

**Review by tommbendall — CHANGES_REQUESTED** [review]

**Review by tommbendall — APPROVED** [review]
Thanks Dan!

**Pull request merged**